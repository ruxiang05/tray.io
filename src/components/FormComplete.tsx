import React from "react";
import { ReactComponent as CheckIcon } from "../assets/check-solid.svg";

const Done = () => (
  <div className="form-complete">
    <CheckIcon className="check-icon" />
    <p>
      Please verify your email address, you should have received an email from
      us already!
    </p>
  </div>
);

export default Done;
