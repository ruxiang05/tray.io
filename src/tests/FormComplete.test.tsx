import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import FormComplete from "../components/FormComplete";

describe("FormComplete", () => {
  it("should render", () => {
    const tree = renderer.create(<FormComplete />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
