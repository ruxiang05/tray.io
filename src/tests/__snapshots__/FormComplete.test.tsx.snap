// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormComplete should render 1`] = `
<div
  className="form-complete"
>
  <svg
    className="check-icon"
  >
    check-solid.svg
  </svg>
  <p>
    Please verify your email address, you should have received an email from us already!
  </p>
</div>
`;
